### PR TITLE
JavaScript source files updated for React and D7.

### PR DESCRIPTION
**Pull request by nmoinvaz**

### CODE REVIEW

File: js/SteamExample.js [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+
+    ob2.release();
+  });
+
+  // Wait for results in notification callbacks
+  steamUserStats.getNumberOfCurrentPlayers();
+  steamUserStats.requestUserStats(mySteamId);
+
+  steamFriends.activateGameOverlayToWebPage("http://google.com/");
+}
```
Comment (steve-tucker) [inline js/SteamExample.js:173]: This is an absurdly long function, might want to think about breaking it up. Also, it's a bit light on the comments for you.

**Review by steve-tucker — APPROVED** [review]

File: js/SteamExample.js (hunk 1 above)
Comment (nmoinvaz) [reply js/SteamExample.js:173]: The name is just copying from Steamworks API. I don't want to change it from what Steamworks provides.

Comment (steve-tucker) [reply js/SteamExample.js:173]: `interopLoaded` - I don't think that's from SteamWorks.

Comment (steve-tucker) [reply js/SteamExample.js:173]: It's 114 lines long, it's got a lot of nugety goodness.

Comment (nmoinvaz) [reply js/SteamExample.js:173]: I think the PR makes it look like I created all that code from scratch, but I did not, I only renamed the file.

**Pull request merged**